climada/hazard/trop_cyclone.py

```diff
@@ -102,26 +102,37 @@ def __init__(self, pool=None):
 
     def set_from_tracks(self, tracks, centroids=None, description='',
                         model='H08', ignore_distance_to_coast=False,
-                        store_windfields=False):
+                        store_windfields=False, metric="equirect"):
```

Comment:
We are going back to "equirect" as the default metric because it's sufficient for 99.99% of the users and 2-3 times faster.